scripts/reducers/bucket.js

```diff
@@ -18,7 +18,10 @@ export function bucket(state = INITIAL_STATE, action) {
       return {...state, busy: action.busy};
     }
     case BUCKET_LOAD_SUCCESS: {
-      const {data} = action;
+      let {bid, data} = action;
+      if (typeof data === "undefined") {
```

Comment:
I think this should be processed in the reducer but rather in the `listBuckets` saga, when we receive the 403.